### PR TITLE
PIM-4866: Added I should see the tab assertion method in behat

### PR DESCRIPTION
**Pull request by ClementGautier**
| Q                  | A
| ------------------ | ---
| Migration script?  | N
| Behats             | N
| Specs & Static     | N
| Changelog updated  | Y

### CODE REVIEW

**Comment by jjanvier** [comment]
GTM but I'm not sure this should appear as a bug fix in the changelog (no idea actually if it was causing troubles or not)

**Comment by ClementGautier** [comment]
@jjanvier Agree, should I use a section like "Technical improvement" ?

File: CHANGELOG-1.3.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 1.3.x
+
+## Technical improvements
+- PIM-4866: Added I should see that tab assertion method in behat
```
Comment (jmleroux) [inline CHANGELOG-1.3.md]: `Added I should see the tab assertion method in behat` ?

**Comment by jmleroux** [comment]
GTM

**Pull request merged**